### DISCUSSION TITLE
Dashboard is empty for all repos after onboarding

### DISCUSSION THREAD

**Discussion by vmaerten** [renovatebot/renovate#21556] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I did the onboarding, renovate bot did the first PR, I am able to connect to the dashboard and see my repos but there is no job in it.
I was expecting to see one here (as I have one renovate PR)
cf : 
![image](https://user-images.githubusercontent.com/9110126/232522008-a3d42529-9dd8-40ec-86c2-791ceb9365e0.png)
![image](https://user-images.githubusercontent.com/9110126/232522125-0abb546f-8665-4488-8a69-bc749fc99a93.png)

EDIT; we have a private org

### What you tried so far.

I tried to remove all cookies and local storage

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Sorry, we have a problem with the dashboard right now. Here is a pinned discussion you can follow: https://github.com/renovatebot/renovate/discussions/21569

**Comment by vmaerten** [comment]
Thanks!
